### ISSUE TITLE
Add `EditorBrowsableState` to advanced FitbitClient constructors

### ISSUE THREAD

**Issue by mxa0079** [aarondcoleman/Fitbit.NET#116]
Let's hide advance mode constructors through this attribute: https://msdn.microsoft.com/en-us/library/system.componentmodel.editorbrowsablestate.aspx

**Comment by WestDiscGolf** [comment]
I don't think that is a good idea, if we hide them then no one will find them and there will be no use in having them :-1: Discovering a new library and API through intellisense is half the way to determining what you can use

**Comment by mxa0079** [comment]
I agree. Do you know a way to influence they way in which Intellisense displays constructors?

**Comment by WestDiscGolf** [comment]
No. As part of the constructor summary descriptions though which gets displayed when intellisense kicks in we could add "Advanced - <description / proposed usages>" so people know that it's a more advanced one?

**Comment by aarondcoleman** [comment]
yeah, let's not hide. Typically, the order is the fewer parameters first in intellisense, so I think we've got that covered. We can add comment headers to that one to make it clear it's more advanced.
